src/libs/actions/PersonalDetails.js

```diff
@@ -101,6 +107,9 @@ function formatPersonalDetails(personalDetailsList) {
  * Get the timezone of the logged in user
  */
 function fetchTimezone() {
+    if (isOffline) {
+        return;
+    }
```

Comment:
NAB, to clarify, I meant to put this in the `setInterval()` - although they effectively do the same thing. If someone removes the reconnect callback they might be confused about why personal details are not queueing while offline.